test/fail_compilation/fail20538.d

```diff
@@ -0,0 +1,14 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail20538.d(12): Error: assignment must be preceded by an identifier
+fail_compilation/fail20538.d(12): Error: found `1` when expecting `,`
```

Comment:
This error message suggests that a = is a valid enum member

Reply:
I don't think so, but I'm open to suggestions.